spec/models/guide_import_spec.rb

```diff
@@ -25,6 +25,10 @@
           teacher_info: 'an info',
           language: 'gobstones',
           solution: 'foo',
+          assistance_rules: [
+            {when: :content_empty, then: 'remember to copy the code in the editor!'},
```

Comment:
We need to also add an editor for those rules in bibliotheca. Otherwise, this PR is pointless.

Reply:
See https://github.com/mumuki/mumuki-bibliotheca/pull/152
See https://github.com/mumuki/mumuki-bibliotheca-api/pull/110